src/constants/index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   AFRAME_INJECTED: 'aframe-injected',
   DEFAULT_CAMERA_HEIGHT: 1.6,
+  DEFAULT_HAND: 'right',
```

Comment:
the property is called `hand`, not sure it helps to make the constant have different name than the property for which it is the default value